include/wx/msw/webview_ie.h

```diff
@@ -146,7 +163,11 @@ class WXDLLIMPEXP_WEBVIEW wxWebViewIE : public wxWebView
 
     // Establish sufficiently modern emulation level for the browser control to
     // allow RunScript() to return any kind of values.
-    static bool MSWSetModernEmulationLevel(bool modernLevel = true);
+    static bool MSWSetModernEmulationLevel(wxWebViewIE_EmulationLevel modernLevel = wxWEBVIEWIE_EMU_IE8);
```

Comment:
The default parameter should be removed, otherwise there is an ambiguity when just `MSWSetModernEmulationLevel()` is called.

Reply:
I rename the method to `MSWSetEmulationLevel()` in 2e7a0151e6b72880fd1154f8ef8f39e1adfe2cde  , because "Modern" dose not suitable for ones like IE7 that must older than a decade.